exporter/humioexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# Humio Exporter
+Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
+
+Supported pipeline types: traces (with metrics and logs to follow soon)
+
+> :construction: This exporter is currently intended for evaluation purposes only!
+
+## Getting Started
+This exporter requires the following configuration options:
+
+- `ingest_token` (no default): The token that has been issued in relation to the Humio repository to export data into. This token grants write-only access to a single, specific Humio repository. See [Ingest Tokens](https://docs.humio.com/docs/ingesting-data/ingest-tokens/) for more details.
+- `endpoint` (no default): The base URL on which the Humio backend can be reached, in the form `host:port`.
```

Comment:
Would each user have their own URLs? An example here could be useful to them.

Reply:
Humio user can either use the SAAS solution, for which everyone shares a single URL, or an on-premises deployment. Humio uses the ingest token to determine the repository to drop the data into. I will make this clearer!

Reply:
Wouldn't it make it more usable then for the SaaS users to have this default to this single URL?

Reply:
Yes, I presume this would be better. Perhaps I should have also been more precise. There are actually two URLs: one for the EU cloud and one for the US cloud (but within those, all users share that single URL).

I could make one of them be the default, and just list the other. Would you have any preference as to one of them?

Reply:
No, if you can't make a good default for the majority of your users, it's fine to not have a default. Perhaps just mention the two most common URLs here?

Reply:
Alright, I will list the URLs and leave the default empty.